spec/features/settings/ext_cert_feature_spec.rb

```diff
@@ -0,0 +1,217 @@
+require "rails_helper"
+
+# rubocop:disable RSpec/ExampleLength
+describe "Feature: External Cerificate settings", js: true do
+
+  let!(:user) { create(:user) }
+  let!(:fixture_path) { RSpec.configuration.fixture_path }
+
+  let(:ssl_cert_file_a) { File.join(fixture_path, "ext_cert_ssl_a.pem") }
+  let(:ssl_key_file_a) { File.join(fixture_path, "ext_cert_key_a.pem") }
+  let(:ssl_cert_file_b) { File.join(fixture_path, "ext_cert_ssl_b.pem") }
+  let(:ssl_key_file_b) { File.join(fixture_path, "ext_cert_key_b.pem") }
+  let(:ssl_cert_file_malformed) { File.join(fixture_path, "ext_cert_ssl_mal.pem") }
+  let(:ssl_key_file_malformed) { File.join(fixture_path, "ext_cert_key_mal.pem") }
+  let(:expired_cert) { File.join(fixture_path, "expired_cert.pem") }
+  let(:key_for_expired_cert) { File.join(fixture_path, "key_for_expired_cert.pem") }
+
+  before do
+    setup_done
+    login_as user, scope: :user
+  end
+
+  describe "#index" do
+    before do
+      visit settings_ext_cert_index_path
+    end
+
+    # Success Conditions
+
+    it "barely does the bare minimum" do
+      expect(page).to have_http_status(:success)
+      expect(page).to have_content("External Certificates", wait: 1)
```

Comment:
You can remove all the `wait: 1`.

Reply:
The current wait time on RSpec fails is terribly long right now.  Obviously not a problem when there are no failures though.  Is there a reason the global wait value is so long?

Reply:
This is some legacy decision when we had issues writing capybara specs. We are safe to to reduce the global timeout to 10-30 seconds.

Reply:
OK.